src/EditorFeatures/CSharpTest/NavigateTo/NavigateToTests.cs

```diff
@@ -1386,6 +1386,32 @@ public class Inner {}
                 },
                 await _aggregator.GetItemsAsync("Outer"));
         }
+
+        [Fact]
+        public async Task DoIncludeSymbolsFromSourceGeneratedFiles()
+        {
+            using var workspace = TestWorkspace.Create(@"
+<Workspace>
+    <Project Language=""C#"" CommonReferences=""true"">
+        <DocumentFromSourceGenerator>
+            public class C
+            {
+            }
+        </DocumentFromSourceGenerator>
+    </Project>
+</Workspace>
+", composition: EditorTestCompositions.EditorFeatures);
+
+            _provider = new NavigateToItemProvider(workspace, AsynchronousOperationListenerProvider.NullListener);
+            _aggregator = new NavigateToTestAggregator(_provider);
+
+            VerifyNavigateToResultItems(
+                new()
```

Comment:
I have no idea what this type is...

Reply:
Nor do I. I don't like our use of target-typed new. :frowning:

(this is copy/pasted from the prior test...so I'm entirely serious when I don't know.)